src/main/java/com/rakovets/course/java/core/practice/decision_making_statements/Task03.java

```diff
@@ -35,8 +35,18 @@ public static void main(String[] args) {
      * @return null/'RED'/'ORANGE'/'YELLOW'/'GREEN'
      */
     static String getColorHealthPoint(int currentHealthPoint, int maxHealthPoint) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
+        if (currentHealthPoint <= (maxHealthPoint * 25 / 100)) {
+            return "RED";
+        }
+        else if (currentHealthPoint <= (maxHealthPoint * 50 / 100)) {
+            return "ORANGE";
+        }
+        else if (currentHealthPoint <= (maxHealthPoint * 75 / 100)) {
+            return "YELLOW";
+        }
+        else if (currentHealthPoint <= maxHealthPoint) {
+            return "GREEN";
+        }
```

Comment:
"Violation: There isn't separating white spaces (OR line break) for curly brake
Solution: Add white spaces"